assets/js/modules/adsense/components/dashboard/DashboardSummaryWidget.js

```diff
@@ -38,23 +38,45 @@ import AdBlockerWarning from '../common/AdBlockerWarning';
 
 const { useSelect } = Data;
 
+/**
+ * Calculates the absolute change between two values.
+ *
+ * @since n.e.x.t
+ *
+ * @param {number} previous The previous value.
+ * @param {number} current  The current value.
+ * @return {number} The absolute change between values.
+ */
```

Comment:
As a local helper, I think the doc block probably isn't necessary. It's basically a 1 line function anyways.